File: lab/python/calculadora/calculadora_exercicio2.py
while True:
    n1, n2 = input().replace(",", ".").split()
    if n1 == "!" or n2 == "!":
        break
    try:
        n1 = float(n1)
        n2 = float(n2)

        soma = n1 + n2
        subtracao = n1 - n2
        multiplicacao = n1 * n2
        divisao = n1 / n2 if n2 != 0 else "Division Error"

        # print("Soma:", soma)
        # print("Subtração:", subtracao)
        # print("Multiplicação:", multiplicacao)
        # print("Divisão:", divisao)
        print(
            f"Soma: {soma}\
                \nSubtração: {subtracao}\
                \nMultiplicação: {multiplicacao}\
                \nDivisão: {divisao}"
            )
    except ValueError:
        print("Inválido, digite apenas números")

File: lab/javascript/calculadora/calculadora_exercicio2.js
const prompt = require("prompt-sync")();

while (true) {
    let entrada = prompt("Digite dois números (ou '!' para sair): ").replace(",", ".");
    let [n1, n2] = entrada.split(" ");

    if (n1 === "!" || n2 === "!") {
        break;
    }

    try {
        n1 = parseFloat(n1);
        n2 = parseFloat(n2);

        if (isNaN(n1) || isNaN(n2)) {
            throw new Error("Inválido, digite apenas números");
        }

        let soma = n1 + n2;
        let subtracao = n1 - n2;
        let multiplicacao = n1 * n2;
        let divisao = (n2 !== 0) ? n1 / n2 : "Division Error";

        console.log(
            `Soma: ${soma}\n` +
            `Subtração: ${subtracao}\n` +
            `Multiplicação: ${multiplicacao}\n` +
            `Divisão: ${divisao}`
        );
    } catch (err) {
        console.log(err.message);
    }
}
